script/misc/update-tag2link

```diff
@@ -0,0 +1,60 @@
+#!/usr/bin/env ruby
+
+#
+# Run with ./script/misc/update-tag2link config/tag2link.yml
+#
+
+require 'net/http'
+require 'json'
+require 'psych'
+
+QUERIES = {
```

Comment:
If possible, add location to original queries in JOSM source code, so it's easier to keep them in sync.

Reply:
done